recipes/haven/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "haven" %}
+{% set version = "0.0.1" %}
+{% set python_min = "3.10.1" %}  # https://github.com/jonmorton/haven/blob/2e501f2db2557bf62cd69c86e2fac1b2280e5770/pyproject.toml#L19
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/jonmorton/haven/archive/2e501f2db2557bf62cd69c86e2fac1b2280e5770.tar.gz
```

Comment:
This url is likely to cause issues for our regro bot. It would be good if there was an official tagged release up on github.

Reply:
Unfortunately, the upstream repository does not have a tag. An issue has been created for this: https://github.com/jonmorton/haven/issues/11